README.md

```diff
@@ -138,8 +138,10 @@ The `action` is one of `PUSH`, `REPLACE`, or `POP` depending on how the user got
 * `history.goBack()`
 * `history.goForward()`
 * `history.canGo(n)` (only in `createMemoryHistory`)
+* `history.link(path, [state])`
```

Comment:
I think `push` & `replace` should go down, or `link` should go up, no?

Reply:
Go up I think is right. Done.